src/System.Runtime/tests/System/Runtime/CompilerServices/ConditionalWeakTableTests.cs

```diff
@@ -205,52 +216,68 @@ public static void AddRemove_DropValue()
             GC.KeepAlive(key);
         }
 
-        [Fact]
-        public static void GetOrCreateValue()
+        [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
+        static void GetWeakRefPair(out WeakReference<object> key_out, out WeakReference<object> val_out)
         {
             var cwt = new ConditionalWeakTable<object, object>();
             var key = new object();
-            object obj = null;
 
             object value = cwt.GetOrCreateValue(key);
 
             Assert.True(cwt.TryGetValue(key, out value));
             Assert.Equal(value, cwt.GetValue(key, k => new object()));
 
-            var wrValue = new WeakReference<object>(value, false);
-            var wrkey = new WeakReference<object>(key, false);
-            key = null;
-            value = null;
+            val_out = new WeakReference<object>(value, false);
+            key_out = new WeakReference<object>(key, false);
+        }
+
+        [Fact]
+        public static void GetOrCreateValue()
+        {
+            object obj = null;
```

Comment:
Nit: this could be declared below just before it's used, and it doesn't need to be initialized to null